package/target_init.gradle

```diff
@@ -54,6 +54,7 @@ allprojects {
 	def buildDir = new File(System.getenv("runelite.pluginhub.package.buildDir"));
 
 	task runelitePluginHubPackage(type: Jar) {
+		duplicatesStrategy = DuplicatesStrategy.INCLUDE
```

Comment:
We're not adding this so one plugin will build, especially if you don't understand what this change is or why it's needed. You should research the root cause of your plugin's build issue and resolve that instead.